### PR TITLE
fix: sideEffects: false cause css file not bundled (fixes #10190)

### LINKED ISSUE #10190: 3.4.2 "sideEffects: false" option causes site to render improperly
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Version
3.4.2

### Environment
webpack 4.6.0

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/jnw46698m3)

### Steps to reproduce
With 3.4.2 I notice antd components CSS doesn't load on my production site. If I edit node_modules/antd/package.json and remove the line "sideEffects: false" it goes back to normal.

My build makes use of babel-plugin-import, MiniCssExtractPlugin, less-loader to build the css

The development site with hot-reloading works fine, but when i build the production site it looks like no antd css loads at all

excerpt from webpack.prod.js
```
{
        test: /\.(css|less)$/,
        use: [
          MiniCssExtractPlugin.loader,
          'css-loader',
          {
            loader: 'less-loader',
            options: {
              modifyVars: antdTheme,
              javascriptEnabled: true,
            },
          },
        ],
      },
```

with 3.4.1 a "yarn build" will produce a vendors.js and vendors.css, but with 3.4.2 the vendors.css does not get generated at all

### What is expected?
The production build worked in 3.4.1 and also works if i remove "sideEffects: false" from node_modules/antd/package.json

### What is actually happening?
unknown

---

### PR DESCRIPTION
**Pull request by aladdin-add**
First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [x] Make sure that you propose PR to right branch: bugfix for `master`, feature for latest active branch `feature-x.x`.
* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

### CODE REVIEW

**Comment by yutingzhao1991** [comment]
close #10190.

File: package.json [hunk 1]
```diff
@@ -212,5 +212,7 @@
   "pre-commit": [
     "lint-staged"
   ],
-  "sideEffects": false
+  "sideEffects": [
+    "es/**/style/*"
```
Comment (whtsky) [inline package.json]: what about styles under `lib/` and `dist/`?

Comment (yutingzhao1991) [reply package.json]: This problem only exist when use tree shaking with es6 module, so not need to include `lib` ant `dist`.

Comment (whtsky) [reply package.json]: `babel-import-plugin` imports from `lib` folder so there's still no css bundled in 3.4.3

Comment (yutingzhao1991) [reply package.json]: let me see, @whtsky do you have any repo for test.

Comment (whtsky) [reply package.json]: I don't have any public repo for this, but I can confirm there's still no css in our project with
`['import', { libraryName: 'antd', style: true }]` config.

I have already opened #10219 for this

**Pull request merged**